guides/source/webpacker.md

```diff
@@ -137,7 +139,7 @@ If you are using a CSS framework, you can add it to Webpacker by following the i
 ### Using Webpacker for Static Assets
 
 The default Webpacker [configuration](https://github.com/rails/webpacker/blob/master/lib/install/config/webpacker.yml#L21) should work out of the box for static assets.
-The configuration includes several image and font file format extensions, allowing Webpack to include them in the generated `manifest.json` file.
+The configuration includes several image and font file format extensions, allowing webpack to include them in the generated `manifest.json` file.
```

Comment:
"webpack" is spelled without a capital everywhere else in this doc.

Reply:
I see mostly capitalized usages outside of commands in the current doc. Please revert this change and let's capitalize it everywhere where it isn't a command (like we do with Active Record).

Reply:
Just to confirm: "Webpacker" is indeed capitalized everywhere in this doc, but I had the impression webpack's official spelling is not.
Even in [its official documentation](https://github.com/webpack/webpack), as the first word of a sentence, it is not capitalized.

Reply:
🤦 I read this as webpacker the first time. My mistake, this makes sense, thanks for clarifying 😄